src/v2/Apps/getAppNovoRoutes.tsx

```diff
@@ -71,14 +72,25 @@ export function getAppNovoRoutes(): RouteConfig[] {
       {
         routes: debugRoutes,
       },
-    ].map((routeConfig: { disabled: boolean; routes: RouteConfig[] }) => {
-      return {
-        ...routeConfig,
-        routes: routeConfig.routes.map(route => ({
-          ...route,
-          path: `/novo${route.path}`,
-        })),
+    ].map(
+      (routeConfig: {
+        disabled?: boolean
+        routes: RouteConfig[]
+        converted?: boolean
+      }) => {
+        return {
+          ...routeConfig,
+          routes: routeConfig.routes.map(route => {
+            if (routeConfig.converted) {
+              return { ...route }
+            }
+            return {
+              ...route,
+              path: `/novo${route.path}`,
```

Comment:
Any chance this `novo`-prefixed path makes its way into analytics events or other places where it might interfere?

Reply:
No more than it already would have.